api/public/cerbos/engine/v1/engine.proto

```diff
@@ -163,6 +163,18 @@ message CheckOutput {
   map<string, ActionEffect> actions = 3;
   repeated string effective_derived_roles = 4;
   repeated cerbos.schema.v1.ValidationError validation_errors = 5;
+  repeated OutputEntry outputs = 6;
+}
+
+message OutputEntry {
+  string src = 1 [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+    description: "Rule that matched to produce this output."
+    example: "\"resource.expense.v1/acme#rule-001\""
+  }];
+  string val = 2 [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
```

Comment:
I wonder whether we should make `val` a protobuf `Value` so that it can hold anything. I think some users might prefer to directly consume native JSON/protobuf from the API response without having to do a second pass to unmarshal the string response.